data/json/furniture_and_terrain/furniture-medical.json

```diff
@@ -10,7 +10,7 @@
     "move_cost_mod": -1,
     "coverage": 35,
     "required_str": -1,
-    "flags": [ "TRANSPARENT", "AUTODOC", "CONTAINER", "NO_PICKUP_ON_EXAMINE" ],
```

Comment:
Remove documentation about this flag as well.

Reply:
Oh damn, just searched the json and src folders. Properly deleting stuff is hard.